lib/events-reader.js

```diff
@@ -5,7 +5,8 @@ var _ = require('lodash'),
     BSON = require('mongodb').BSONPure,
     mongojs = require('mongojs'),
     debounce = require('debounce-promise'),
-    hl = require('highland');
+    hl = require('highland'),
+    checkpointWriter = require('./checkpoint-writer');
```

Comment:
might be good to call the file events-reader-checkpoint-writer or something, since there are checkpoints for sse as well